MethodDecorator.Fody.Tests/TestsBase.cs

```diff
@@ -16,8 +16,8 @@ public abstract class TestsBase : IDisposable {
 
         protected IList<Tuple<Method, object[]>> Records {
             get {
-                var records = (IList<Tuple<int, object[]>>)this.RecordHost.Records;
```

Comment:
Will it be better if you add a framework version preprocessor for such code? Possible implementation can be found here http://stackoverflow.com/questions/4535622/can-i-make-a-preprocessor-directive-dependent-on-the-net-framework-version